common/predictive-text/worker/word_breaking/placeholder-word-breaker.ts

```diff
@@ -0,0 +1,24 @@
+namespace wordBreakers {
+
+  /**
+   * A **VERY** dumb word breaker that simply splits at words. Do not use this
+   * word breaker!
+   *
+   * @param phrase The phrase in which to break words.
+   * @deprecated Use a word breaker tailored to your language instead!
+   */
+  export function placeholderWordBreaker(phrase: string): Span[] {
```

Comment:
Based on https://github.com/keymanapp/lexical-models/pull/27/, this should probably be just `placeholder`?

Reply:
In reference to the following points:
* [`wordBreakingSource = "wordBreakers['${modelSource.wordBreaking}']";`](https://github.com/keymanapp/lexical-models/pull/27/files#diff-a937db9fe61b7df3dce08173c065a25cR112)  (Tweaked quotes due to formatting issues)
  * This appears to be aimed at accessing members of the `wordBreakers` namespace provided by this PR.
* [Type specs for lexical models](https://github.com/keymanapp/lexical-models/pull/27/files#diff-309e71a2a0e19d27b0ef70e159ade219R28)
  * Note - `placeholder` there vs `placeholderWordBreaker` here.

Reply:
Nice catch! You are absolutely correct; it should export the name `placeholder` and not `placeholderWordBreaker`.